src/devices/video/imagetek_i4100.h

```diff
@@ -59,6 +59,9 @@ class imagetek_i4100_device : public device_t,
 	u32 screen_update(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect);
 	void draw_foreground(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect);
 	DECLARE_WRITE_LINE_MEMBER(screen_eof);
+	auto ext_ctrl_0_cb() { return m_ext_ctrl_0_cb.bind(); }
+	auto ext_ctrl_1_cb() { return m_ext_ctrl_1_cb.bind(); }
+	auto ext_ctrl_2_cb() { return m_ext_ctrl_2_cb.bind(); }
```

Comment:
Configuration member functions should be kept together – not mixed with live functions.

Reply:
Was treating screen_eof as a configuration option, but I'll move it anyway.